docs/en/pid-eaa-issuance.rst

```diff
@@ -237,22 +250,21 @@ The JWS of Request Object is represented below:
 .. note::
 
     The Wallet Instance redirect URI is a universal or app link registered with the local operating system, so this latter will resolve it and pass the response to the Wallet Instance.
- 
+
 .. code-block:: http
 
     HTTP/1.1 302 Found
-    Location: eudiw://start.wallet.example.org?code=SplxlOBeZQQYbYS6WxSbIA&state=fyZiOL9Lf2CeKuNT2JzxiLRDink0uPcd&iss=https%3A%2F%2Fpid-provider.example.org
+    Location: https://start.wallet.example.org?code=SplxlOBeZQQYbYS6WxSbIA&state=fyZiOL9Lf2CeKuNT2JzxiLRDink0uPcd&iss=https%3A%2F%2Fpid-provider.example.org
 
-**Steps 12-13 (DPoP Proof for Token Endpoint)**: The Wallet Instance MUST create a new key pair for the DPoP and a fresh DPoP Proof JWT following the instruction provided in Section 4 of (:rfc:`9449`) for the token request to the PID/(Q)EAA Provider. The DPoP Proof JWT is signed using the created private key for DPoP by Wallet Instance. DPoP provides a way to bind the Access Token to a certain sender (Wallet Instance) (:rfc:`9449`). This mitigates the misuse of leaked or stolen Access Tokens at the Credential Endpoint of PID/(Q)EAA Issuer as the attacker needs to present a valid DPoP Proof JWT.
+**Steps 12-13 (DPoP Proof for Token Endpoint)**: The Wallet Instance MUST create a new key pair for the DPoP and a fresh DPoP Proof JWT following the instruction provided in Section 4 of (:rfc:`9449`) for the token request to the PID/(Q)EAA Provider. The DPoP Proof JWT is signed using the private key for DPoP created by Wallet Instance for this scope. DPoP binds the Access Token to a certain sender (Wallet Instance) (:rfc:`9449`) and mitigates the misuse of leaked or stolen Access Tokens at the Credential Endpoint of PID/(Q)EAA Issuer.
 
-**Step 14 (Token Request):** The Wallet Instance sends a token request to the PID/(Q)EAA Provider Token Endpoint using the authorization ``code``, ``code_verifier``, *DPoP Proof JWT* and OAuth 2.0 Attestation based Client Authentication parameters (``client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-client-attestation`` and ``client_assertion=WIA~WIA-PoP``). 
-The ``client_assertion`` is signed using the private key that is created during the setup phase to obtain the Wallet Instance Attestation. The related public key that is attested by the Wallet Provider is provided within the Wallet Instance Attestation (``cnf`` claim). The PID/(Q)EAA Provider performs the following checks on the Token Request:
```

Comment:
well, considering that at this endpoint the client uses the code, obtained after a client and an user authentication and also a code challenge (that's a secret!), I'd keep the WIA~POP at protection of the authorization endpoint without requiring it again here.

@asharif1990 ^

Reply:
So, I would recommend keeping it here in any case as an additional security layer. It is right before releasing the Access Token for the credential endpoint so keeping the Wallet Authentication would not harm I would say. The Wallet Authentication at the PAR endpoint is an early-stage mitigation, while the authentication at the token endpoint is a good practice to be kept and to be compliant with both Core specs and HAIP. WDYT?

Reply:
I think that I'll keep it as it is with WIA~POP at the token endpoint to not increase the noise, with the awareness that we have discussed this and that we'll continue exploring this way with other authors to get an agreement

Reply:
text restored and comment resolved, @asharif1990 ^